index.html

```diff
@@ -127,6 +127,36 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ Samir Contributor card END ________  -->
 
+        <!-- Saurabh Contributor Card start -->
+        <div class="card">
+          <p class="name">Vipul Varshney</p>
+          <p class="contact">
+              <i class="fab fa-instagram"></i>
+              <a href="https://www.instagram.com/sau.ra.bh7" target="_blank">Hit me up on gram</a>
+              <i class="fab fa-github"></i>
+              <a href="https://www.github.com/saurabh1118" target="_blank">Here's my GitHub</a>
+          </p>
+          <p class="about">Hello, I am a MERN stack developer, DSA geek and a cybersecuirty enthusiast.</p>
+          <div class="resources">
+              <p>3 Useful Dev Resources</p>
+              <ul>
+                  <li>
+                      <a href="https://www.freecodecamp.org/news/the-css-flexbox-handbook/" target="_blank"
+                          title="First Resource">Free Code Camp CSS Flex Box Tutorial book</a>
+                  </li>
+                  <li>
+                      <a href="https://https://developer.mozilla.org/" target="_blank" title="Second Resource">MDN Web Dev
```

Comment:
Hi, you had an extra `https://` in this link. You should update it by committing and pushing again. The we'll review it.
Thank you for your contribution

```suggestion
                      <a href="https://developer.mozilla.org/" target="_blank" title="Second Resource">MDN Web Dev
```